dask/dataframe/io/parquet.py

```diff
@@ -821,6 +822,8 @@ def to_parquet(df, path, engine='auto', compression='default', write_index=None,
 
     fs, fs_token, _ = get_fs_token_paths(path, mode='wb',
                                          storage_options=storage_options)
+    # Trim any protocol information from the path before forwarding
+    path = infer_storage_options(path)['path']
```

Comment:
I'm not certain this is right, because for s3 paths, the bucket part looks like the "server" in a normal URL.

Reply:
This was fixed in #3069. See https://github.com/dask/dask/blob/master/dask/bytes/utils.py#L65-L72.

Reply:
I stand corrected